cluster-provision/k8s/1.18/provision.sh

```diff
@@ -263,19 +268,32 @@ certificatesDir: /etc/kubernetes/pki
 clusterName: kubernetes
 controllerManager:
   extraArgs:
-    feature-gates: BlockVolume=true,CSIBlockVolume=true,VolumeSnapshotDataSource=true
+    cluster-cidr: 10.244.0.0/16,fd20:0:2::/64
+    feature-gates: BlockVolume=true,CSIBlockVolume=true,VolumeSnapshotDataSource=true,IPv6DualStack=true
+    node-cidr-mask-size-ipv4: "16"
+    node-cidr-mask-size-ipv6: "64"
+    service-cluster-ip-range: 10.96.0.0/12,fd00:10:96::/112
```

Comment:
Same with services subnet, 
Could you change the services ipv6 CIDR to `fd10:96::/108` so it will be more human friendly?
pod network subnet will be like:
```
fd10:96:: ~ fd10:96::f:ffff
instead of:
fd00:10:96:: ~ fd00:10:96::ffff
```
108 prefix will make 65536 available address (like 16 prefix on ipv4)

Reply:
not in this PR please if possible, there is no need to block for it. we need to advance and can advance things in parallel.
lot of refactor and fixing was done already.

Reply:
Since this is the first time we have ipv6 CIDR this is not fall under refactoring, but proper configurations.
We should follow the principal I presented in https://github.com/kubevirt/kubevirt/pull/3328 
where you set easy to read ipv6 address for better experience and debugging like what we did for VM's ipv6 CIDR.
Yes we discussed about this on your demo presenting this and you agreed this should be done (only one person said this is not mandatory w/o understanding the problem).

Like I said this PR looks good you already did great job, it is important to configure these kind of things in place and not postponing them for later.
I am not saying this whats going to happen but look at the https://github.com/kubevirt/kubevirt/pull/3328 discussion about  the outcome of changing CIDR after it was already defined, it could lead to issues with upgrades and VM migrations.

If you insist to not fix it in this PR then let do it at least before bumping kubevirtci in kubevirt.

Reply:
done

Reply:
Thank you :)